manifests/apache/conf.pp

```diff
@@ -1,86 +1,37 @@
-# == Class: puppetboard::apache::conf
-#
-# Creates an entry in your apache configuration directory
-# to run PuppetBoard server-wide (i.e. not in a vhost).
-#
-# === Parameters
-#
-# Document parameters here.
-#
-# [*wsgi_alias*]
-#   (string) WSGI script alias source
-#   Default: '/puppetboard'
-#
-# [*threads*]
-#   (int) Number of WSGI threads to use.
-#   Defaults to 5
-#
-# [*max_reqs*]
-#   (int) Limit on number of requests allowed to daemon process
-#   Defaults to 0 (no limit)
-#
-# [*user*]
-#   (string) WSGI daemon process user, and daemon process name
-#   Defaults to 'puppetboard' ($::puppetboard::params::user)
-#
-# [*group*]
-#   (int) WSGI daemon process group owner, and daemon process group
-#   Defaults to 'puppetboard' ($::puppetboard::params::group)
-#
-# [*basedir*]
-#   (string) Base directory where to build puppetboard vcsrepo and python virtualenv.
-#   Defaults to '/srv/puppetboard' ($::puppetboard::params::basedir)
-#
-# [*enable_ldap_auth]
-#   (bool) Whether to enable LDAP auth
-#   Defaults to False ($::puppetboard::params::enable_ldap_auth)
-#
-# [*ldap_bind_dn]
-#   (string) LDAP Bind DN
-#   No default ($::puppetboard::params::ldap_bind_dn)
-#
-# [*ldap_bind_password]
-#   (string) LDAP password
-#   No default ($::puppetboard::params::ldap_bind_password)
-#
-# [*ldap_url]
-#   (string) LDAP connection string
-#   No default ($::puppetboard::params::ldap_url)
-#
-# [*ldap_bind_authoritative]
-#   (string) Determines if other authentication providers are used when a user can be mapped to a DN but the server cannot bind with the credentials
-#   No default ($::puppetboard::params::ldap_bind_authoritative)
-#
-# [*ldap_require_group]
-#   (bool) LDAP group to require on login
-#   Default to False ($::puppetboard::params::ldap_require_group)
-#
-# [*$ldap_require_group_dn]
-#   (string) LDAP group DN for LDAP group
-#   No default
-#
-# === Notes:
-#
-# Make sure you have purge_configs set to false in your apache class!
-#
-# This runs the WSGI application with a WSGIProcessGroup of $user and
-# a WSGIApplicationGroup of %{GLOBAL}.
+# @summary Creates an entry in your apache configuration directory to run PuppetBoard server-wide (i.e. not in a vhost).
+#
+# @param wsgi_alias WSGI script alias source
+# @param threads Number of WSGI threads to use
+# @param max_reqs Limit on number of requests allowed to daemon process Defaults to 0 (no limit)
+# @param user WSGI daemon process user, and daemon process name
+# @param group WSGI daemon process group owner, and daemon process group
+# @param basedir Base directory where to build puppetboard vcsrepo and python virtualenv.
+# @param enable_ldap_auth Whether to enable LDAP auth
+# @param ldap_bind_dn LDAP Bind DN
+# @param ldap_bind_password LDAP password
+# @param ldap_url LDAP connection string
+# @param ldap_bind_authoritative Determines if other authentication providers are used when a user can be mapped to a DN but the server cannot bind with the credentials
```

Comment:
I often write this as
```suggestion
# @param
#  ldap_bind_authoritative Determines if other authentication providers are used when a user can be mapped to a DN but the server cannot bind with the credentials
```

And then keep a max line length. Thoughts on this?

Reply:
sounds good!

Reply:
The parameter name should come after `@param` though, not on the line underneath...